webClient/src/app/app.component.ts

```diff
@@ -69,6 +69,23 @@ export class AppComponent {
     if (this.launchMetadata != null && this.launchMetadata.data != null && this.launchMetadata.data.type != null) {
       this.handleLaunchOrMessageObject(this.launchMetadata.data);
     }
+
```

Comment:
Ignore the following for cluster support testing 

(This is testing for the logger messages substitution due to my unclean branch)